lms/static/js/groups/views/cohort_discussions_course_wide.js

```diff
@@ -29,13 +29,15 @@
                      * @returns {HtmlSnippet} - HTML list for course-wide discussion topics.
                      */
                     getCourseWideDiscussionsHtml: function(courseWideDiscussions) {
-                        var subCategoryTemplate = HtmlUtils.template($('#cohort-discussions-subcategory-tpl').html()),
+                        var entryTemplate = HtmlUtils.template($('#cohort-discussions-subcategory-tpl').html()),
```

Comment:
@muzaffaryousaf: I changed and moved the comment above to a [Glossary in Confluence](https://openedx.atlassian.net/wiki/display/TNL/Discussion+Modules+and+Categories#DiscussionModulesandCategories-Glossary) on a related page.  Feel free to comment/edit there instead of here.

@attiyaIshaque: This move shouldn't affect my recommendations above.  I just got started on bullet 1.